src/fragments/guides/storage/transfer-accelaration.mdx

```diff
@@ -0,0 +1,43 @@
+When you need leverage S3's transfer accelaration on the content bucket, you can override the storage resource for this configuration and then leveraging the `useAccelerateEndpoint` parameter to use the accelarated S3 endpoint. This will enable fast, easy, and secure transfers of files over long distances between your client and the S3 bucket.
```

Comment:
You can leverage transfer acceleration to enable fast and secure transfer of files over long distances between your end user device and the S3 bucket. You can override the storage resource for this configuration and then leverage the `useAccelerateEndpoint` parameter to use the accelerated S3 endpoint.

Reply:
Modified the guide wording as per this feedback